### PR TITLE
Resource publish

### PR DESCRIPTION
**Pull request by kdimopulu**

### CODE REVIEW

File: staff_features/resources/step_definitions/resource_shared.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  expect(find('#resource_publish_').checked?).to eq true
+end
+
+Given 'the {string} button is displayed' do |button_text|
+  expect(page).to have_css('a', text: button_text)
+end
+
+Then 'the Resource opens on a new tab in the public interface' do
+  expect(page.windows.size).to eq 2
+  expect(page).to have_text "Resource #{@uuid}"
```
Comment (brianzelip) [inline staff_features/resources/step_definitions/resource_shared.rb:18]: Can we be more specific here to test for the public url pattern, something like `#{PUI_URL}/repositories/#{REPO_ID}/resources/#{RESOURCE_ID}`? (this comment duplicates [my comment](https://github.com/archivesspace/e2e-tests/pull/38#discussion_r1892245778) left in PR #38)

**Review by brianzelip — CHANGES_REQUESTED** [review]

**Review by balmas — COMMENTED** [review]
looks ok, agree with Brian's request

File: staff_features/resources/step_definitions/resource_shared.rb (hunk 1 above)
Comment (blacksmith-welder) [reply staff_features/resources/step_definitions/resource_shared.rb:18]: Yeah right. Thanks for pointing this out Brian. Done.

**Review by brianzelip — APPROVED** [review]
👍🏼

**Pull request closed without merging**